projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -5,7 +5,7 @@
 use Automattic\Jetpack\Sync\Settings;
 
 class Jetpack_RelatedPosts {
-	const VERSION   = '20210930';
+	const VERSION   = '20210931';
```

Comment:
You can use today's date instead:

```suggestion
	const VERSION   = '20211209';
```

Reply:
@jeherve, done. Thank you so much. 

Q: In the future, if I just update JavaScript files, but not CSS,  I wouldn't have to bump the version, I reckon?

Reply:
It mostly depends on how the files are enqueued. When enqueuing a file in WordPress, you usually use the `wp_enqueue_*` functions. They allow you to specify a version number that's appended the resource's path as a query string, acting as a cache buster:

<img width="1312" alt="Screenshot 2021-12-09 at 14 05 31" src="https://user-images.githubusercontent.com/426388/145401898-0f60be3b-1c24-4a78-b262-3a888b42e360.png">

In most Jetpack files, we use a constant to define the version, `JETPACK__VERSION`. That constant is updated every time we release a new version of Jetpack, so we don't have to worry about refreshing it, it's done with each new release.
Here is how Jetpack Likes's stylesheet is loaded, for example:
https://github.com/Automattic/jetpack/blob/4d6d490fb8f8f62a947917f080c33c95c4c8b2a6/projects/plugins/jetpack/modules/likes.php#L156

Related posts, however, use their own value for historical reasons:
https://github.com/Automattic/jetpack/blob/edcc988c4e422fd3609ab00c80d2c5cb57b1f95b/projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php#L1704

This is why we have to update it here.

Reply:
Thank you for the detailed explanation. That makes sense.